arviz/plots/traceplot.py

```diff
@@ -240,3 +291,45 @@ def _histplot_op(ax, data, **kwargs):
     bins = get_bins(data)
     ax.hist(data, bins=bins, align="left", density=True, **kwargs)
     return ax
```

Comment:
This function should probably be deleted. It looks like kde/hist is handled by plot_dist

Reply:
good catch!